driver/main_args.ml

```diff
@@ -135,9 +135,16 @@ let mk_for_pack_opt f =
 let mk_g_byt f =
   "-g", Arg.Unit f, " Save debugging information"
 
+let mk_no_g_byt f =
+  "-no-g", Arg.Unit f, " Do not save debugging information (default)"
+
 let mk_g_opt f =
   "-g", Arg.Unit f, " Record debugging information for exception backtrace"
 
+let mk_no_g_opt f =
+  "-no-g", Arg.Unit f,
+  " Do not record debugging information for exception backtrace (default)"
```

Comment:
Is there a reason to have a different wording for bytecode and native? What does it refer to, anyway? I guess it means that debugging information in bytecode contains typing information as well as backtrace descriptors. Personally, I would get rid of the difference and just use `Record/save debugging information` for both.